src/__tests__/mutationResults.ts

```
@@ -685,9 +679,10 @@ describe('mutation results', () => {
         },
       })).then(() => {
         subscriptionHandle.unsubscribe();
-        expect(errors).toHaveLength(1);
-        expect(errors[0].message).toBe(`Hello... It's me.`);
-        console.error = oldError;
+        reject("should have thrown");
+      }, error => {
+        subscriptionHandle.unsubscribe();
+        expect(error.message).toBe(`Hello... It's me.`);
```

These tests were failing silently because of the way the `tryFunctionOrLogError` function was turning exceptions into console errors. In other words, eliminating `tryFunctionOrLogError` is important to ensure these tests will actually report future regressions in the way exceptions are handled for mutation `update` functions.